tests/drivers/test_zarr_driver.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
+from typing import Tuple
+
+import numpy as np
+import pytest
+import zarr
+
+from hydromt.drivers.zarr_driver import ZarrDriver
+
+
+class TestZarrDriver:
+    @pytest.fixture()
+    def example_zarr_file(self, tmp_dir: Path) -> Tuple[zarr.Array, Path]:
+        tmp_path: Path = tmp_dir / "0s.zarr"
```

Comment:
Does this work in 3.9? I coudln't find when this was added.

Reply:
same as above, pathlib, using a dunder method to make a path more readable